### PR TITLE
fix: (Platform) Input Group accessibility

### PR DESCRIPTION
**Pull request by Betrozov**
#### Please provide a link to the associated issue.
https://github.com/sap/fundamental-ngx/issues/3492
#### Please provide a brief summary of this pull request.
This PR contains fixes for the accessibility issues for the Input Group component.
#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [X] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [X] update `README.md`
- [X] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [X] Documentation Examples
- [X] Stackblitz works for all examples

### CODE REVIEW

**Review by mykolavorobiov — APPROVED** [review]

**Review by KevinOkamoto — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-compact-example.component.html [hunk 1]
```diff
@@ -1,10 +1,15 @@
+<label fd-form-label for="input-group-compact" id="input-group-compact-label">Compact</label>
 <fdp-input-group
+    [id]="'input-group-compact'"
```
Comment (olshansky) [inline apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-compact-example.component.html:3]:
```suggestion
    id="input-group-compact"
```

File: apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-disabled-example.component.html [hunk 2]
```diff
@@ -1,10 +1,15 @@
+<label fd-form-label for="input-group-disabled" id="input-group-disabled-label">Disabled</label>
 <fdp-input-group
+    [id]="'input-group-disabled'"
```
Comment (olshansky) [inline apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-disabled-example.component.html:3]:
```suggestion
    id="input-group-disabled"
```

**Review by olshansky — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-compact-example.component.html (hunk 1 above)
Comment (Betrozov) [reply apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-compact-example.component.html:3]: @olshansky I would love to do it, but it doesn't work

**Comment by puru-hk** [comment]
Tested looks fine

**Review by puru-hk — APPROVED** [review]

**Pull request merged**